src/nvim/buffer_defs.h

```diff
@@ -958,6 +958,33 @@ struct matchitem {
   int conceal_char;         ///< cchar for Conceal highlighting
 };
 
+typedef enum {
+    kFloatAnchorEast = 1,
+    kFloatAnchorSouth = 2,
+
+    kFloatAnchorNW = 0,
+    kFloatAnchorNE = 1,
+    kFloatAnchorSW = 2,
```

Comment:
`kFloatAnchorEast`/`kFloatAnchorSouth` also use 1/2. Can't `win_config_float()` use `kFloatAnchorNE`/`kFloatAnchorSW` instead?

Reply:
It doesn't mean the same thing, one are flags, the other are combinations of flags. Though we could use only concrete flags in code like `kFloatAnchorEast | kFloatAnchorSouth` and only keep dummy combination item for compatibility. Or not use `enum` storage for values, rather typedef an int.